backend/spec/features/admin/orders/new_order_spec.rb

```diff
@@ -175,8 +170,6 @@
     end
 
     it "transitions to delivery not to complete" do
-      click_on 'Cart'
-
```

Comment:
Can we have these changes to the new order and customer details feature specs as operate PRs to master? I want to be sure they still pass on non-Turbolinks based backend.